packages/ra-data-graphql-simple/src/buildVariables.test.ts

```diff
@@ -173,3 +173,181 @@ describe('buildVariables', () => {
         });
     });
 });
+
+describe('buildVariables with meta param', () => {
```

Comment:
Great job with these tests! 👍

However, I suggest putting them in the existings and correspondings `describe` (e.g: the `GET_LIST` should go in the existing `GET_LIST` in the existing `describe('buildVariables')`)
Additionally, since the purpose of these tests is to test `meta` and `sparseFields` in particular, I suggest renaming them with a more explicit name and reducing their scope as described in the following suggestions.